docs/etcd/backup-restore.md

```diff
@@ -58,8 +57,11 @@ etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/m
 etcd-manager-ctl --backup-store=s3://my.clusters/test.my.clusters/backups/etcd/events restore-backup [events backup file]
 ```
 
-Note that this does not start the restore immediately; you need to restart etcd on all masters 
-(or roll your masters quickly). A new etcd cluster will be created and the backup will be 
+Note that this does not start the restore immediately; you need to restart etcd on all masters.
+You can do this with a `docker stop` or `kill` on the etcd-manager containers on the masters (the container names start with `k8s_etcd-manager_etcd-manager`).
```

Comment:
you can always add the `docker restart` for the pods

Reply:
> you can always add the `docker restart` for the pods

In my mind, stopping them makes more sense, just letting the containers get back on their own. And it's also the way @justinsb told me to do it, if I remember correctly. A restart should technically work too though.